src/main/resources/lang/default.lang

```diff
@@ -975,6 +975,9 @@ entities:
 	zombie villager:
 		name: zombie villager¦s
 		pattern: <age> zombie villager(|1¦s)
+	zombie normal:
+		name: villager¦s
```

Comment:
I'm assuming you forgot something here?
id assume this should be like `zombie villager` or `zombie normal`